tests/Zend/Session/SessionTest.php

```diff
@@ -1091,6 +1095,8 @@ public function testInvalidPreexistingSessionIdDoesNotPreventRegenerationOfSid()
             @mkdir($sessionStore . DIRECTORY_SEPARATOR . $subdir);
         }
 
+        session_start();
```

Comment:
This particular test requires that there be an existing session running, but due to the changes in the `tearDown` method above, this wasn't the case any longer. Forcing a start here (after the save_path is changed and the directory it's going to write to is created).